salt/elife-dashboard/config/srv-elife-dashboard-2-uwsgi.ini

```diff
@@ -0,0 +1,25 @@
+{% set app = pillar.elife_dashboard_2 %}
+[uwsgi]
+chdir=/srv/elife-dashboard/dashboard_2/
+uid={{ pillar.elife.webserver.username }}
+gid={{ pillar.elife.webserver.username }}
+pythonpath=/srv/elife-dashboard/dashboard_2/
```

Comment:
this can probably be removed as we removed it from `profiles`

Reply:
Ok, removed.